design/nab_status_update.md

```diff
@@ -0,0 +1,111 @@
+# Developer Workflow: NonAdminBackup Phase and Conditions within NonAdminBackup Status
+
+## Overview
+
+This document outlines the design around updating NonAdminBackup objects Phase and Conditions within Status.
+
+### Phase
+
+A NonAdminBackup's `status` field has a `phase` field, which is updated by NAC controller.
+
+The `phase` is a simple one high-level summary of the lifecycle of an NonAdminBackup.
+
+It is always a one well defined value, that is intended to be a comprehensive state of a NonAdminBackup.
+
+Those are are the possible values for phase:
+
+| **Value** | **Description**                 |
+|-----------|--------------------------------|
+| New | *NonAdminBackup resource was accepted by the OpenShift cluster, but it has not yet been processed by the NonAdminController* |
+| BackingOff | Velero *Backup* object was not created due to NonAdminBackup error (configuration or similar) |
+| Created | Velero *Backup* was created. The Phase will not have additional informations about the |
```

Comment:
about the.... :question: :question:

Reply:
`Velero *Backup* was created. The Phase will not have additional informations about the *Backup* run`